context.go

```diff
@@ -46,6 +46,9 @@ type Context struct {
 	// The currently used WebSocket subprotocol by the client, set on a successful handshake.
 	activeSubProtocol string
 
+	// Regular expressions that the Origin header must match (if non-empty)
```

Comment:
Not regular expressions

```go
	// Each pattern is matched case insensitively against the request origin host
	// with filepath.Match.
	// See https://golang.org/pkg/path/filepath/#Match
```

https://github.com/nhooyr/websocket/blob/e3a2d32f704fb06c439e56d2a85334de04b50d32/accept.go#L43-L45

```suggestion
	// Patterns that the Origin header must match (if non-empty)
```